src/client/utils/html.ts

```diff
@@ -315,8 +315,14 @@ export function processHtml (html, options: ProcessHTMLOptions = {}) {
 
         if (!parentTag) {
             if (htmlElements.length) {
-                for (const htmlElement of htmlElements)
-                    nativeMethods.insertAdjacentHTML.call(htmlElement, 'afterbegin', INIT_SCRIPT_FOR_IFRAME_TEMPLATE);
+                for (const htmlElement of htmlElements) {
+                    const firstScriptOrStyle = nativeMethods.elementQuerySelector.call(htmlElement, 'script,link[rel="stylesheet"]');
```

Comment:
Create a constant on the module level:
const SCRIPT_AND_STYLE_SELECTOR = 'script,link[rel="stylesheet"]';